cylc/rose/entry_points.py

```diff
@@ -218,8 +212,7 @@ def record_cylc_install_options(
     dumper.dump(cli_config, str(conf_filepath))
 
     # Merge the opts section of the rose-suite.conf with those set by CLI:
-    if not rose_conf_filepath.is_file():
-        rose_conf_filepath.touch()
+    rose_conf_filepath.touch()
```

Comment:
Not sure about this one. The next line loads this file so if it is always a newly created empty file what would be the point of loading it?

Reply:
https://github.com/cylc/cylc-rose/pull/196/files#r1033651643